tesseract/tesseract_collision/include/tesseract_collision/bullet/bullet_collision_object.h

```diff
@@ -0,0 +1,121 @@
+#ifndef TESSERACT_COLLISION_BULLET_COLLISION_OBJECT_H
+#define TESSERACT_COLLISION_BULLET_COLLISION_OBJECT_H
+
+#include <tesseract_collision/core/collision_object_base.h>
+#include <tesseract_core/macros.h>
```

Comment:
@Levi-Armstrong I believe this needs to be changed to tesseract_collision/core/macros.h. Same for types